frontend/src/pages/booking/utils/phoneNumberValidator.ts

```diff
@@ -0,0 +1,16 @@
+import { ERROR_MESSAGE } from '../constants/errorMessage';
+
+const validatePhoneNumberLength = (phoneNumber: string) => {
+  if (phoneNumber === '') return '';
+
+  const digits = phoneNumber.replace(/\D/g, '');
+  return digits.length !== 11;
```

Comment:
p4: 11을 상수화 하면 어떨까? `VALID_PHONE_NUMBER_LENGTH` 🤔
```suggestion
  return digits.length !== 11;
```